sql/src/test/java/org/opensearch/sql/sql/antlr/SQLSyntaxParserTest.java

```diff
@@ -193,6 +193,12 @@ public void can_parse_now_like_functions(String name, Boolean hasFsp, Boolean ha
 
   @Test
   public void can_parse_multi_match_relevance_function() {
+    assertNotNull(parser.parse(
+        "SELECT id FROM test WHERE multi_match(\"fields\"=\"field\", \"query\"=\"query\")"));
+    assertNotNull(parser.parse(
+        "SELECT id FROM test WHERE multi_match(\'fields\'=\'field\', \'query\'=\'query\')"));
+    assertNotNull(parser.parse(
+        "SELECT id FROM test WHERE multi_match(fields=\'field\', query=\'query\')"));
```

Comment:
Should there also be a test for `multimatchquery`?

Reply:
Fix: [a722d51af36e88d3c0c6eb2ce5a69392d2011695](https://github.com/Bit-Quill/opensearch-project-sql/pull/168/commits/a722d51af36e88d3c0c6eb2ce5a69392d2011695)